psi-ms.obo

```diff
@@ -22655,6 +22655,386 @@ relationship: has_metric_category MS:4000012 ! single run based metric
 relationship: has_value_type xsd:int ! The allowed value-type for this CV term
 relationship: has_units UO:0000189 ! count unit
 
+[Term]
+id: MS:4000108
+name: Peak density distribution MS1 Q1, Q2, Q3
```

Comment:
**suggestion:** Should we rename this as "quantiles" and dynamically determine the number of splits from the values, as is the case for a few other terms already? In that case term `MS:4000109` would maybe be superfluous, because it would be the distribution consisting of only a single value?

Reply:
Not impossible, but it would be good if we discuss the ways we handle that type of information for downstream purposes.

Reply:
As discussed in telco, now quantiles, a visualisation guide to the metrics to follow (outside obo)